inst/css/sidebar.css

```diff
@@ -26,13 +41,16 @@ a.disabled {
   flex-direction: row;
   align-items: center;
 }
+
 .manager_table_row *:first-child {
   flex: 1 1 80%;
 }
-.manager_table_row * + * {
+
+.manager_table_row *+* {
   flex: 0 0 0px;
   padding: 0em 1.5em;
 }
+
 .manager_placeholder {
   margin-top: 1em;
-}
+}
```

Comment:
What is this? :thinking: 
![image](https://github.com/insightsengineering/teal/assets/114988527/83407515-c995-4b27-afa2-236f05fffc54)